schema/bitbucket_server.schema.json

```
@@ -107,6 +107,12 @@
               "description": "Secret for authenticating incoming webhook payloads",
               "type": "string",
               "minLength": 1
+            },
+            "automaticCreation": {
```

```suggestion
            "automaticSync": {
```

Since it also updates the webhook, I think that makes more sense, no?

---

Another thought: since the default value is "true" anyway, why not make this a negative flag, i.e. `disableSync`?

That to me makes more sense than a default being set without me able to see it in the config.

---

I almost did it that way but thought that a negative flag was more confusing. But I guess it makes sense that they should explicitly opt out if the default is to sync